test/evergreen.yml

```diff
@@ -923,6 +923,19 @@ tasks:
 
   # End of Python unit test tasks
 
+  - name: s-all
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/dist"
+          script: |
+            set -o errexit
+            set -o verbose
+            sh s_all -A -E 2>&1
```

Comment:
What's the benefit of `-A -E` options?

Reply:
To make it fail if something is wrong otherwise the command will always exit with exit code 0.